File: csharp-netstandard/src/Cloud.Governance.Client/Model/DynamicRuleCondition.cs


using System;
using System.Collections;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.IO;
using System.Runtime.Serialization;
using System.Text;
using System.Text.RegularExpressions;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using System.ComponentModel.DataAnnotations;
using OpenAPIDateConverter = Cloud.Governance.Client.Client.OpenAPIDateConverter;

namespace Cloud.Governance.Client.Model
{
    
    [JsonConverter(typeof(StringEnumConverter))]
    
    public enum DynamicRuleCondition
    {
        [EnumMember(Value = "0")]
        Equals = 0,

        [EnumMember(Value = "1")]
        Contains = 1,

        [EnumMember(Value = "2")]
        Is = 2,

        [EnumMember(Value = "3")]
        In = 3,

        [EnumMember(Value = "4")]
        DoesNotBelongTo = 4

    }

}

File: typescript/src/models/DynamicRuleCondition.ts
/* tslint:disable */
/* eslint-disable */
/*
 * Cloud Governance Api
 *
 * Contact: support@avepoint.com
 */

/**
 * 
 * @export
 * @enum {string}
 */
export enum DynamicRuleCondition {
    Equals = 0,
    Contains = 1,
    Is = 2,
    In = 3,
    DoesNotBelongTo = 4
}

export function DynamicRuleConditionFromJSON(json: any): DynamicRuleCondition {
    return DynamicRuleConditionFromJSONTyped(json, false);
}

export function DynamicRuleConditionFromJSONTyped(json: any, ignoreDiscriminator: boolean): DynamicRuleCondition {
    return json as DynamicRuleCondition;
}

export function DynamicRuleConditionToJSON(value?: DynamicRuleCondition | null): any {
    return value as any;
}

